src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_001_02_Export.scala

```diff
@@ -118,6 +118,20 @@ class Pain008_001_02_Export extends LazyLogging {
   private def getSumAllRechnungs(rechnungen: List[Rechnung]): BigDecimal = {
     rechnungen.map(_.betrag).sum
   }
+
+  def randomStringFromCharList(length: Int, chars: Seq[Char]): String = {
+    val sb = new StringBuilder
+    for (i <- 1 to length) {
+      val randomNum = util.Random.nextInt(chars.length)
+      sb.append(chars(randomNum))
+    }
+    sb.toString
+  }
+
+  def randomAlphaNumericString(length: Int): String = {
+    val chars = ('a' to 'z') ++ ('A' to 'Z') ++ ('0' to '9')
+    randomStringFromCharList(length, chars)
+  }
 }
```

Comment:
you can replace that with:
scala.util.Random.alphanumeric.take(lenght).mkString